scanpipe/models.py

```diff
@@ -1471,6 +1467,41 @@ def has_value(self, field_name):
 class ScanFieldsModelMixin(models.Model):
     """Fields returned by the ScanCode-toolkit scans."""
 
+    detected_license_expression = models.TextField(
+        blank=True,
+        help_text=_(
+            "The license expression summarizing the license info for this resource, "
+            "combined from all the license detections"
+        ),
+    )
+    detected_license_expression_spdx = models.TextField(
+        blank=True,
+        help_text=_(
+            "The detected license expression for this file, with SPDX license keys"
+        ),
+    )
+    license_detections = models.JSONField(
+        blank=True,
+        default=list,
+        help_text=_("List of license detection details."),
+    )
+    license_clues = models.JSONField(
+        blank=True,
+        default=list,
+        help_text=_(
+            "List of license matches that are not proper detections and potentially "
+            "just clues to licenses or likely false positives. Those are not included "
+            "in computing the detected license expression for the resource."
+        ),
+    )
+    percentage_of_license_text = models.FloatField(
+        blank=True,
+        null=True,
+        help_text=_(
```

Comment:
Let's use this instead (I am updating SCTK too):

```suggestion
        help_text=_("Percentage of file words detected as license text or notice."),
```